Fetch data from WakaTime + Invite employees/managers to organisation

nikkaroraa opened the pull request.

nikkaroraa committed:
[Feat]: Fetch userDetails, projects and userActivity from WakaTime

anshurathee commented on src/helpers/fetchWakatimeUserDetails.js:
```diff
@@ -0,0 +1,42 @@
+      return createErrorMessage('Caught an error while making API request to WakaTime');
+    }
+    switch (e.response.status) {
+      case 400:
```
Put all these in a constants file.

nikkaroraa committed:
[Feat]: Implement feature/organisation and refactoring

1. List Organisations
2. Implement magic-link functionality to invite/add employees and managers

anshurathee commented on src/views/inviteMail.ejs:
```diff
@@ -0,0 +1,11 @@
+<% } %>
+<br />
+
+Click <a href="http://localhost:8080/invite/register?token=<%= token %>"</a>here</a> to register.
```
Use dynamic links
